src/main/resources/swagger/api-docs.yaml

```diff
@@ -1986,6 +1986,31 @@ paths:
           required: false
           schema:
             $ref: '#/definitions/AttributeUpdateOperation'
+
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/{entityName}/evaluate:
+      post:
+        tags:
+          - Entities
+        operationId: getEntity
+        summary: Get entity in a workspace
+        parameters:
+          - $ref: '#/parameters/workspaceNamespaceParam'
+          - $ref: '#/parameters/workspaceNameParam'
+          - $ref: '#/parameters/entityTypeParam'
+          - $ref: '#/parameters/entityNameParam'
+          - in: body
+            description: Expression
+            name: expression
+            required: true
+            type: string
+        responses:
+          200:
+            description: Success
+          404:
```

Comment:
Ninja-ed comment on Rawls not specifying a 403 for this. Are we treating 403's as 404's in the current endpoint on purpose for security reasons?